Formula/curl.rb

```diff
@@ -22,24 +23,29 @@ class Curl < Formula
   keg_only :provided_by_macos
 
   depends_on "pkg-config" => :build
+  depends_on "libressl"
```

Comment:
Doesn't macOS come with libreSSL?

Reply:
Yes, but doesn't work

Reply:
I mean, Apple got it to work so there must be a way 😄

Reply:
`uses_from_macos` + `Formula["libressl"].opt_prefix` = ?

Reply:
> To be clear: Apple does _not_ support linking against system LibreSSL. They don't provide the headers necessary to compile against it, so LibreSSL would have to come from Homebrew.

Reply:
@SeekingMeaning But you still can build curl with Apple's SSL/TLS by using `--with-darwinssl` option.